app/Http/Controllers/FaqController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Faq;
+use Illuminate\Http\Request;
+
+class FaqController extends Controller
+{
+    public function create()
+    {
+        $menu = MainController::menu('faq');
```

Comment:
It is not good to need to call another controller from within another one. What you are solving I think is that you want to have one common menu where the active menu item is dynamic. It can be solved in an easier way in your Layout, or even with a Blade component if you want. You might check out the TaskItEasy code for some inspiration.